### ISSUE TITLE
Support for returning status message in trailers

### ISSUE THREAD

**Issue by freynder** [eclipse-vertx/vertx-grpc#51]
#### Describe the feature
The current implementation only implements `grpc-status` currently. The [protocol](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) allows `grpc-message` as well.

#### Use cases
Any gRPC call could benefit from it. Especially useful during manual testing and logging.

**Comment by vietj** [comment]
do you mind contributing this @freynder

**Comment by freynder** [comment]
Hi @vietj 
That was my original intention, however I got stuck at the part where there was only a `GrpcStatus` and not the full Status object available to retrieve the value. I did not continue due to lack of time.
I'll have another look at it in the weekend and let you know. 
Are you ok with using the `PercentEscaper` from (already included) Guava lib (https://guava.dev/releases/19.0/api/docs/com/google/common/net/PercentEscaper.html) ? The message should be escaped and io.grpc uses its own private routines for this which are not accessible to us.
